pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/service/download/DownloadParticipantListServiceTest.java

```diff
@@ -58,45 +66,84 @@ public static void cleanUp() {
     }
 
     @Test
-    public void testDataDownloadFromElastic() {
+    public void testKitRequestShippingDownloadFromElastic() {
         ManualFilterParticipantList filterable = getFilterFromFile("elastic/filterWithSampleQueueColumn.json");
         Assert.assertNotNull(filterable);
         QueryParamsMap queryParamsMap = buildMockQueryParams(true, true, "xlsx");
         List<ParticipantWrapperDto> downloadList = DownloadParticipantListService.fetchParticipantEsData(filterable, queryParamsMap);
-        assertDownloadList(downloadList);
+        assertKitRequestShipping(downloadList);
 
         queryParamsMap = buildMockQueryParams(false, true, "xlsx");
         downloadList = DownloadParticipantListService.fetchParticipantEsData(filterable, queryParamsMap);
-        assertDownloadList(downloadList);
+        assertKitRequestShipping(downloadList);
 
         queryParamsMap = buildMockQueryParams(true, false, "xlsx");
         downloadList = DownloadParticipantListService.fetchParticipantEsData(filterable, queryParamsMap);
-        assertDownloadList(downloadList);
+        assertKitRequestShipping(downloadList);
 
         queryParamsMap = buildMockQueryParams(false, false, "xlsx");
         downloadList = DownloadParticipantListService.fetchParticipantEsData(filterable, queryParamsMap);
-        assertDownloadList(downloadList);
+        assertKitRequestShipping(downloadList);
 
         queryParamsMap = buildMockQueryParams(true, true, "tsv");
         downloadList = DownloadParticipantListService.fetchParticipantEsData(filterable, queryParamsMap);
-        assertDownloadList(downloadList);
+        assertKitRequestShipping(downloadList);
 
         queryParamsMap = buildMockQueryParams(false, true, "tsv");
         downloadList = DownloadParticipantListService.fetchParticipantEsData(filterable, queryParamsMap);
-        assertDownloadList(downloadList);
+        assertKitRequestShipping(downloadList);
 
         queryParamsMap = buildMockQueryParams(true, false, "tsv");
         downloadList = DownloadParticipantListService.fetchParticipantEsData(filterable, queryParamsMap);
-        assertDownloadList(downloadList);
+        assertKitRequestShipping(downloadList);
 
         queryParamsMap = buildMockQueryParams(false, false, "tsv");
         downloadList = DownloadParticipantListService.fetchParticipantEsData(filterable, queryParamsMap);
-        assertDownloadList(downloadList);
+        assertKitRequestShipping(downloadList);
+    }
+
+    @Test
+    public void testClinicalOrdersDownloadFromElastic() {
+        ManualFilterParticipantList filterable = getFilterFromFile("elastic/filtersWithClinicalOrderColumns.json");
+        List<Filter> columnNames = getColumnNames("elastic/filtersWithClinicalOrderColumns.json");
```

Comment:
Yes because one extracts the filters and one the display columns and in the main code both of them come from the request (which is in the same shape of the contents of that file)